Connectors/Autocad/Speckle.Connectors.AutocadShared/HostApp/AutocadMaterialBaker.cs

```diff
@@ -264,4 +128,60 @@ public List<ReceiveConversionResult> ParseAndBakeRenderMaterials(
     transaction.Commit();
     return results;
   }
+
+  private (ObjectId, ReceiveConversionResult) BakeMaterial(
+    RenderMaterial renderMaterial,
+    string baseLayerPrefix,
+    DBDictionary materialDict,
+    Transaction tr
+  )
+  {
+    ObjectId materialId = ObjectId.Null;
+
+    try
+    {
+      // POC: Currently we're relying on the render material name for identification if it's coming from speckle and from which model; could we do something else?
+      // POC: we should assume render materials all have application ids?
+      string renderMaterialId = renderMaterial.applicationId ?? renderMaterial.id;
+      string matName = _autocadContext.RemoveInvalidChars(
+        $"{renderMaterial.name}-({renderMaterialId})-{baseLayerPrefix}"
+      );
+
+      MaterialMap map = new();
+      MaterialOpacityComponent opacity = new(renderMaterial.opacity, map);
+      var systemDiffuse = System.Drawing.Color.FromArgb(renderMaterial.diffuse);
+      EntityColor entityDiffuseColor = new(systemDiffuse.R, systemDiffuse.G, systemDiffuse.B);
+      MaterialColor diffuseColor = new(Method.Override, 1, entityDiffuseColor);
+      MaterialDiffuseComponent diffuse = new(diffuseColor, map);
+
+      Material mat =
+        new()
+        {
+          Name = matName,
+          Opacity = opacity,
+          Diffuse = diffuse
+        };
+
+      if (renderMaterial["reflectivity"] is double reflectivity)
+      {
+        mat.Reflectivity = reflectivity;
+      }
+
+      if (renderMaterial["ior"] is double ior)
```

Comment:
Something perhaps we should just add to `RenderMaterial`
Not needed for this PR tho